ui/src/app/pages/data/concept/concept-add-modal.tsx

```diff
@@ -31,6 +25,16 @@ const styles = reactStyles({
   }
 });
 
+const filterConcepts = (concepts: any[], domain: Domain) => {
```

Comment:
Will, does PM still have a separate bucket at the bottom of the UI?

Reply:
Yes, the code is there but is currently feature flagged. Should only appear in local and test environments